vue-frontend/src/components/services/WhatWeOfferSection.vue

```diff
@@ -88,22 +97,20 @@ export default {
             "tw-bg-gradient-to-b tw-from-[#E3D9EE]  via-[#E3D9EE]  tw-to-white tw-bg-gradient-[180deg] ",
         },
         {
-          images: [seo400w, seo800w],
-          title: "SEO / Digital Marketer",
+          images: [backend400w, backend800w],
+          title: "Backend Development",
           description:
-            "To scale your business exponentially you have to get organic reach for your business and we know how to do it. In just one year, \
-            we grew from 16 daily impressions to 24369 daily impressions. 1500X impressions growth under one year. We know how it works and what it takes.",
+            "The 'behind-the-scenes' backbone of any app, the backend, handles the critical tasks of data management, server configuration, and application logic. Our experts ensure your app is robust, secure, and efficient from its core, enabling it to handle high traffic loads, process data swiftly, and interact flawlessly with front-end requests.",
           class:
-            "tw-bg-gradient-to-b tw-from-[#EBEDF3]  via-[#EBEDF3]  tw-to-white tw-bg-gradient-[180deg] ",
+            "tw-bg-gradient-to-b tw-from-[#001035] via-[#001035] tw-to-white tw-bg-gradient-[180deg] ",
```

Comment:
Navy blue color looks unpleasant for mobile/tablet resolution. Let's raise this point in channel.